examples/integration/Earthfile

```diff
@@ -0,0 +1,78 @@
+FROM docker:19.03.7-dind
```

Comment:
Should we call the directory `examples/integration-test`? I'm thinking it's more suggestive if somebody is glancing at the contents of the `examples` dir.